addons/gatekeeper/gatekeeper.yaml

```
@@ -33,7 +33,7 @@ spec:
   chartReference:
     chart: gatekeeper
     repo: https://mesosphere.github.io/charts/staging
-    version: 0.6.5
+    version: 0.6.6
```

```suggestion
    version: 0.7.0
```
Lets follow the process. Bump the minor version when bumping the app minor version.

---

ah, i gotta change the chart then....